### ISSUE TITLE
CTRL + F Highlights

### ISSUE THREAD

**Issue by karl-police** [KingSora/OverlayScrollbars#459]
These things 
![image](https://user-images.githubusercontent.com/12023782/189536296-d68924c7-ecb1-40f2-b6e4-ed72318568a0.png)

which are present on Google Chrome as example, when you use CTRL + F to highlight where results were found. Unfortunately the JavaScript version doesn't have it because it is something entirely different

Google Chrome does have an overlay scrollbar flag, but it's not in JS, though does have those things still

I am not sure if javascript can track that CTRL + F window, there might be hacky ways to do it, but what about a good way?

The only way I can imagine, is to prevent the default function of CTRL + F and instead override it with a custom search window

I don't know why in css-drafts they don't wanna change it, so that the overlay scrollbars can be done just with CSS again:

https://github.com/w3c/csswg-drafts/issues/6090

https://github.com/w3c/csswg-drafts/issues/7716

**Comment by KingSora** [comment]
Good day @karl-police

I'm afraid there is no way I could add this easily.. This is a limitation of all scrollbar plugins since this feature relies on the native browser scrollbar.

**Comment by karl-police** [comment]
> Good day @karl-police
> 
> I'm afraid there is no way I could add this easily.. This is a limitation of all scrollbar plugins since this feature relies on the native browser scrollbar.

Yeah I thought so. I mean maybe if a CTRL + F window gets recoded and then add that what Google Chrome would be doing. Drawing those rectangles would be questionable though, like with fixed positions or something idk.

I hope CSSWG-Drafts will give scrollbar-gutter the ability to overlay over the page. But I don't think it's going to happen any soon.
